refinery/analysis_manager/models.py

```diff
@@ -73,9 +73,6 @@ def refinery_import_state(self):
         return get_task_group_state(self.refinery_import_task_group_id)
 
     def galaxy_file_import_state(self):
-        return get_task_group_state(self.galaxy_import_task_group_id)
-
-    def tool_based_galaxy_file_import_state(self):
         if self.galaxy_import_state and self.galaxy_import_progress != 0:
```

Comment:
Yes, you can still construct an AnalysisStatus like so: `AnalysisStatus.objects.create(analysis=analysis)`

Reply:
Right but the field would actually exists with a default value, no?

Reply:
The default value for a `blank=True` is `u''` which is still falsey

For example, this test case passes:
```
def test_analysis_status_galaxy_import_state_is_falsey_by_default(self):
        analysis_status = AnalysisStatus.objects.create(analysis=self.analysis)
        self.assertFalse(analysis_status.galaxy_import_state)
```

Reply:
Yes but the field is there, so checking for its existence is not necessary.

Reply:
Its necessary as far as I can tell. 

One would need to assert that there has been a `galaxy_import_state` set and that some `galaxy_import_progress` has happened before going ahead and handing a response over to the front end to display. 
Otherwise theres nothing meaningful for the UI to display yet.

I could imagine a scenario occurring where the `galaxy_import_state` is still equal to `u''` and even though the `galaxy_import_state` field is present, passing that information along to the frontend without checking it first would yield odd results.

Reply:
OK but you are already checking it by `self.galaxy_import_progress != 0`

Reply:
Just because there is positive `galaxy_import_progress` doesn't mean that a valid `galaxy_import_state` has been set as well?